utils/symbol.c

```diff
@@ -767,6 +770,38 @@ static int update_symtab_using_dynsym(struct symtab *symtab, const char *filenam
 	return ret;
 }
 
+void load_python_symtab(struct symtabs *symtabs)
+{
+	char *symfile = NULL;
+	struct uftrace_mmap *map;
+
+	/* try to load python symtab (if exists) */
+	xasprintf(&symfile, "%s/%s", symtabs->dirname, PYTHON_SYMTAB_NAME);
+	if (access(symfile, R_OK) < 0) {
+		free(symfile);
+		return;
+	}
+
+	/* add a fake map for python script */
+	map = xzalloc(sizeof(*map) + sizeof("<python>"));
+
+	map->start = 0;
+	map->end = 4096;
+	map->len = 8;
+
+	memcpy(map->prot, "rwxp", 4);
+	strcpy(map->libname, "<python>");
+
+	load_module_symbol_file(&map->symtab, symfile, 0);
```

Comment:
This is not possible because `symtab` was removed from `map` by the following commit.
- 56fd7247 symbol: Remove symtab from symtabs and maps

It needs more refactoring.